Overview.bs

```diff
@@ -487,30 +491,26 @@ Media: interactive
 Animatable: no
 </pre>
 
-
 <dl dfn-type=value dfn-for=animator>
     <dt><dfn>none</dfn>
     <dd>
-        There will be no animators.
-    <dt><dfn><<ident>></dfn>
+      This element will not be passed into any <a>animate function</a> as one of the
+      <a>children element proxies</a>.
+    <dt><dfn><<ident>>#</dfn>
     <dd>
-      If there is a <a>animator definition</a> registered with that name, then this element is
-      assigned to the first ancestor animator instance of the given name as a child element proxy.
-      If no <a>animator definition</a> exists with that name, then the element gets assigned as soon
-      as one is registered.
-
-      Issue: Todo: If no ancestor animator instance exists with that name then we should create one
-      with document root element as its root.
-</dl>
+      For each <<ident>> specified in the list, the following applies for each animation frame:
 
+      * If there is no <a>animator definition</a> registered with the given <<ident>>, the inclusion
+        of that <<ident>> as a value has no effect.
 
-Note: All the elements in the root element's DOM sub-tree that get associated with the same animator
- name will get assigned to the the animator instance linked with this root element.
+      * Otherwise, an <a>element proxy</a> for this element will be passed into the
+        <a>animate function</a> of the closest ancestor <a>animator instance</a> with the given
+        <<ident>> as one of the <a>children element proxies</a>. If no such <a>animator instance</a>
+        exists, one is created with the document root element as it's <a>root element proxy</a>.
```

Comment:
Except that I don't think we'll actually pass in a proxy for the document root. This should maybe just refer to the animator instance with no root element?